src/styles/common/reset.styl

```diff
@@ -0,0 +1,20 @@
+html, body, div, span, h1, h2, h3, h4, h5, h6,
+p, pre, a, code, img, i, ol, ul, li,
+fieldset, form, label, article, aside, footer, header,
+menu, nav, section
+  margin: 0
+  padding: 0
+  border: 0
+  font-size: 100%
+  font: inherit
+  vertical-align: baseline
+
+article, aside,
+footer, header, nav, section
+	display: block
```

Comment:
No tabs, please. Use spaces. Let's keep one style across all the files.

Reply:
will fix this